src/protagonist/DLCS.Model/Assets/AssetFilter.cs

```diff
@@ -9,5 +11,6 @@ public class AssetFilter
     public int? NumberReference1 { get; set; }
     public int? NumberReference2 { get; set; }
     public int? NumberReference3 { get; set; }
+    public List<string>? Manifests { get; set; }
```

Comment:
Use `string[]` to match Tags (feels more natural as it won't be altered)? Or update Tags to use `List<string>` ?